boards/shields/mcp2515/doc/index.rst

```diff
@@ -244,7 +244,7 @@ example:
    https://www.dfrobot.com/product-1444.html
 
 .. _DFRobot CAN BUS Shield V2.0 schematic:
-   https://github.com/DFRobot/CAN_BUS/blob/master/DFR0370%20CAN%20BUS%20sheild(V2.0).pdf
+   https://github.com/DFRobot/DFRobot_MCP2515/blob/master/resources/doc/DFR0370%20CAN%20BUS%20sheild(V2.0).pdf
```

Comment:
General question: There are a lot of PDF files (and some images) being references in our docs with links that look like they will get updated automatically (like this one). 

Can we host those file ourselves as an alternative?

Reply:
I think that would be a copyright/license nightmare, not to mention storage implications

Reply:
I think you are right :)